conformance/src/main/scala/za/co/absa/enceladus/conformance/DynamicConformanceJob.scala

```diff
@@ -217,12 +218,8 @@ object DynamicConformanceJob {
       case None    => withPartCols.count
       case Some(p) => p
     }
+    if (recordCount == 0) { handleEmptyOutputAfterConformance }
```

Comment:
```suggestion
    if (recordCount == 0) { handleEmptyOutputAfterConformance() }
```
Needs the parens as it is not an accessor or a field, but a method with side effects (throwing an exception) on top.

Reply:
I think it reads better too